### PR TITLE
bigquery: add simple benchmark

### PR DESCRIPTION
**Pull request by pongad**
Runs queries and measure time to first response
and time to iterate all rows

### CODE REVIEW

**Comment by pongad** [comment]
@lukesneeringer We'll probably start up something similar for spanner soon. According to current design, the benchmarks will be a little more involved though. As we have more benchmarks, I also hope things will become less ad-hoc.

So the answer to your question is: yes, but the "flavor" of the PRs will probably change.

File: bigquery/benchmark/benchmark.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+from google.cloud import bigquery
```
Comment (dhermes) [inline bigquery/benchmark/benchmark.py]: This file needs the `LICENSE` header

**Comment by pongad** [comment]
@dhermes Sorry about that. https://github.com/GoogleCloudPlatform/google-cloud-python/pull/4284

File: bigquery/benchmark/benchmark.py (hunk 1 above)
Comment (dhermes) [reply bigquery/benchmark/benchmark.py]: Thanks

**Pull request merged**